client.go

```diff
@@ -172,7 +170,5 @@ func SetBus(b *partybus.Bus) {
 // Cleanup deletes all directories created by stereoscope calls. Note: please use image.Image.Cleanup() over this
 // function when possible.
 func Cleanup() {
-	if err := rootTempDirGenerator.Cleanup(); err != nil {
-		log.Errorf("failed to cleanup tempdir root: %w", err)
-	}
+	log.Debug("cleanup is deprecated - no need to call. please use image.Image.Cleanup()")
```

Comment:
Deprecation shouldn't change behavior (this would be a breaking change). Instead we should up date the comment.